File: app/src/main/java/org/sil/storyproducer/model/messaging/Message.java
package org.sil.storyproducer.model.messaging;

/**
 * Created by annmcostantino on 4/7/2018.
 */

public class Message {
    private final boolean fromPhone;
    private final String message;

    public Message(boolean b, String m){
        fromPhone = b;
        message = m;
    }

    public boolean isFromPhone(){
        return fromPhone;
    }

    public String getMessage(){
        return message;
    }
}

File: app/src/main/java/org/sil/storyproducer/model/messaging/Message.kt
package org.sil.storyproducer.model.messaging

import java.sql.Timestamp

/**
 * Created by annmcostantino on 4/7/2018.
 */

class Message(
    val slideNumber: Int,
    val storyId: Int,
    val isConsultant: Boolean,
    val isTranscript: Boolean,
    val timeSent: Timestamp,
    val message: String
)
